chart/Chart.yaml

```diff
@@ -47,25 +47,25 @@ annotations:
       url: https://airflow.apache.org/docs/helm-chart/1.8.0/
   artifacthub.io/screenshots: |
     - title: DAGs View
-      url: https://airflow.apache.org/docs/apache-airflow/2.6.2/_images/dags.png
+      url: https://airflow.apache.org/docs/apache-airflow/2.6.3/_images/dags.png
     - title: Datasets View
-      url: https://airflow.apache.org/docs/apache-airflow/2.6.2/_images/datasets.png
+      url: https://airflow.apache.org/docs/apache-airflow/2.6.3/_images/datasets.png
     - title: Grid View
-      url: https://airflow.apache.org/docs/apache-airflow/2.6.2/_images/grid.png
+      url: https://airflow.apache.org/docs/apache-airflow/2.6.3/_images/grid.png
     - title: Graph View
-      url: https://airflow.apache.org/docs/apache-airflow/2.6.2/_images/graph.png
+      url: https://airflow.apache.org/docs/apache-airflow/2.6.3/_images/graph.png
     - title: Calendar View
-      url: https://airflow.apache.org/docs/apache-airflow/2.6.2/_images/calendar.png
+      url: https://airflow.apache.org/docs/apache-airflow/2.6.3/_images/calendar.png
     - title: Variable View
-      url: https://airflow.apache.org/docs/apache-airflow/2.6.2/_images/variable_hidden.png
+      url: https://airflow.apache.org/docs/apache-airflow/2.6.3/_images/variable_hidden.png
     - title: Gantt Chart
-      url: https://airflow.apache.org/docs/apache-airflow/2.6.2/_images/gantt.png
+      url: https://airflow.apache.org/docs/apache-airflow/2.6.3/_images/gantt.png
     - title: Task Duration
-      url: https://airflow.apache.org/docs/apache-airflow/2.6.2/_images/duration.png
+      url: https://airflow.apache.org/docs/apache-airflow/2.6.3/_images/duration.png
     - title: Code View
-      url: https://airflow.apache.org/docs/apache-airflow/2.6.2/_images/code.png
+      url: https://airflow.apache.org/docs/apache-airflow/2.6.3/_images/code.png
     - title: Task Instance Context Menu
-      url: https://airflow.apache.org/docs/apache-airflow/2.6.2/_images/context.png
+      url: https://airflow.apache.org/docs/apache-airflow/2.6.3/_images/context.png
```

Comment:
This Task Instance Context Menu image is no longer resolving correctly.

Reply:
Remove it :)